### ISSUE TITLE
SEO 2024

### ISSUE THREAD

**Issue by nrllh** [HTTPArchive/almanac.httparchive.org#3600]
# SEO 2024

![SEO illustration](https://raw.githubusercontent.com/HTTPArchive/almanac.httparchive.org/main/src/static/images/2019/seo/hero_lg.jpg)

If you're interested in contributing to the CSS chapter of the 2024 Web Almanac, please reply to this issue and indicate which role or roles best fit your interest and availability: [author](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide), [reviewer](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide), [analyst](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide), and/or [editor](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide). 

## Content team

| Lead | Authors | Reviewers | Analysts | Editors | Coordinator |
| ---- | ------- | --------- | -------- | ------- |    -------  |
| - | - | - | - | - | -  | - |

<details>
<summary><b>Expand for more information about each role 👀</b></summary>

- The **[content team lead](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Content-Team-Leads'-Guide)** is the chapter owner and responsible for setting the scope of the chapter and managing contributors' day-to-day progress.
- **[Authors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide)** are subject matter experts and lead the content direction for each chapter. Chapters typically have one or two authors. Authors are responsible for planning the outline of the chapter, analyzing stats and trends, and writing the annual report.
- **[Reviewers](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide)** are also subject matter experts and assist authors with technical reviews during the planning, analyzing, and writing phases.
- **[Analysts](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide)** are responsible for researching the stats and trends used throughout the Almanac. Analysts work closely with authors and reviewers during the planning phase to give direction on the types of stats that are possible from the dataset, and during the analyzing/writing phases to ensure that the stats are used correctly.
- **[Editors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide)** are technical writers who have a penchant for both technical and non-technical content correctness. Editors have a mastery of the English language and work closely with authors to help wordsmith content and ensure that everything fits together as a cohesive unit.
- The **[section coordinator](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Section-Leads'-Guide)** is the overall owner for all chapters within a section like "User Experience" or "Page Content" and helps to keep each chapter on schedule.

_Note: The time commitment for each role varies by the chapter's scope and complexity as well as the number of contributors._

For an overview of how the roles work together at each phase of the project, see the [Chapter Lifecycle](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle) doc.
</details>

### [0. Form the content team](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#0-create-content-team)
- [ ] 📆 `April 15` Complete program and content committee - 🔑 *Organizing committee*
  - The content team has at least one author, reviewer, and analyst.

### [1. Plan content](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#1-plan-content)
- [ ] 📆 `May 1` **First meeting to outline the chapter contents** - 🔑 *Content team*
  - The content team has completed the chapter outline.

### [2. Gather data](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#2-gather-data)
- [ ] 📆 `June 1` **Custom metrics completed** - 🔑 *Analysts*
  - Analysts have added all necessary [custom metrics](https://github.com/HTTPArchive/custom-metrics/blob/main/README.md) and drafted a PR ([example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1087)) to track query progress. 
- [ ]  📆 `June 1` **HTTP Archive Crawl** - 🔑 *HA Team*
  - HTTP Archive runs the June crawl.

### [3. Validate results](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#3-validate-results)
- [ ] 📆 `August 15` **Query Metrics & Save Results** - 🔑 *Analysts*
  - Analysts have queried all metrics and saved the output.

### [4. Draft content](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#4-draft-content)
- [ ] 📆 `September 15` **First Draft of Chapter** - 🔑 *Authors*
  - Authors has written the chapter. 
- [ ] 📆 `October 10` **Review & Edit Chapter** - 🔑 *Reviewers & Editors*
  - Reviewers and  Editors has processed the the chapter.

### [5. Publication](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#5-publication)
- [ ] 📆 `October 15` **Chapter Publication (Markdown & PR)** - 🔑 *Authors*
  - Authors has converted the chapter to markdown and drafted a PR.
- [ ] 📆 `November 1` **Launch of 2024 Web Almanac 🚀** - 🔑 *Organizing committee*
 
### [6. Virtual conference](#)
- [ ] 📆 `November 20` **Virtual Conference** - 🔑 *Content Team*

**Comment by patrickstox** [comment]
I'd be happy to be an author or reviewer again for the SEO chapter.

**Comment by MichaelLewittes** [comment]
And I'd be happy to be the editor of the SEO section again.

**Comment by amandeepsinghvirdi** [comment]
Happy to contribute as Reviewer.

**Comment by AVGP** [comment]
Happy to join as content lead.

**Comment by henryp25** [comment]
I would love to put myself forward as an analyst or author.

**Comment by en3r0** [comment]
Happy to contribute as a reviewer again!

**Comment by dwsmart** [comment]
Be honoured to be either a coauthor or reviewer

**Comment by fellowhuman1101** [comment]
I'd be delighted to return to this chapter as an author or reviewer. Bonus points for a 3rd go coauthoring with @dwsmart

**Comment by guaca** [comment]
Happy to contribute as analyst!

**Comment by nrllh** [comment]
We have another great team here! Thank you all!

**Comment by cnichols013** [comment]
Happy to help as an analyst.

**Comment by nrllh** [comment]
> Happy to help as an analyst.

Thanks!

**Comment by MichaelLewittes** [comment]
I'm traveling internationally on May 1 -- the Plan Content meeting -- but
happy to discuss/meet beforehand (or right afterwards).

**Comment by tomvangoethem** [comment]
It looks like we don't have a lead for this chapter yet. Are any of the authors interested in taking up that role? @patrickstox, @AVGP, @dwsmart, @fellowhuman1101? A first step would be to set up a quick call with the different contributors of this chapters (preferably early this upcoming week) to align with everyone on the outline of the chapter. Thanks!

**Comment by dwsmart** [comment]
@AVGP indicated he wanted to take the lead role above, and I happen to think that is a winning idea.

**Comment by MichaelLewittes** [comment]
Winning idea, indeed!

**Comment by AVGP** [comment]
Yep, still up for it, though I won't be available for the call tomorrow, if that's a problem?

**Comment by dwsmart** [comment]
Perhaps we can do a call on the following Mon, Tue or Wed? Although a few days late, I think that having the content lead there outweighs that. I..m.h.o.

**Comment by MichaelLewittes** [comment]
Agree. What works next week for you Martin?

**Comment by patrickstox** [comment]
I have some travel coming up but will do my best to make it whenever y'all want. Also endorsing  @AVGP for lead.

**Comment by AVGP** [comment]
I could set something up for May 15th?

**Comment by dwsmart** [comment]
That's good for me

**Comment by MichaelLewittes** [comment]
Will be at Google I/O but can work around it, if I’m needed this early.

**Comment by AVGP** [comment]
Anyone who commented here and hasn't been onboarded to the Slack channel and/or the contributor calls yet? Please raise your hands :)

**Comment by en3r0** [comment]
I fit that description!

**Comment by AVGP** [comment]
> I fit that description!

@en3r0 Then please join the slack channel at https://httparchive.slack.com/archives/C021X2E1NHL

**Comment by en3r0** [comment]
Trying to get there, but it says no account found for that channel.

I would prefer to not need to monitor another communication point, is it possible to be notified here or via email when the time comes for the review? I believe that is how it was done in the past.

**Comment by en3r0** [comment]
Never received a notification here or an email to review this.

**Comment by ayres1000** [comment]
Hi, I want to contribute like analyst, author or translator(pt-br)

**Comment by simodepth96** [comment]
I'd be happy to put myself as an analyst for the Web Almanac 2025
@henryp25, @cnichols013
